packages/brick-kit/src/core/Router.ts

```diff
@@ -300,6 +301,8 @@ export class Router {
               type: brickPageError,
               properties: {
                 error: httpErrorToString(error),
+                code:
+                  error instanceof HttpResponseError && error.response?.status,
```

Comment:
这样的话 `code` 可能为 `false`，这不是预期的吧？应该这样：

```suggestion
                  error instanceof HttpResponseError ? error.response.status : null
```